libs/wingsdk/test/target-tf-aws/redis.inflight.test.ts

```diff
@@ -89,36 +87,50 @@ test("can delete a key", async () => {
   expect(value).toBeUndefined();
 });
 
-const fakeRedisClient = {
```

Comment:
Don't worry, my rewriting of this file was [completely unnecessary](https://github.com/winglang/wing/pull/5753#discussion_r1498349268) because everything here was totally fine